internal/providers/terraform/azure/lb_outbound_rule.go

```diff
@@ -0,0 +1,38 @@
+package azure
+
+import (
+	"strings"
+
+	"github.com/infracost/infracost/internal/schema"
+)
+
+func GetAzureRMLoadBalancerOutboundRuleRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "azurerm_lb_outbound_rule",
+		RFunc: NewAzureRMLoadBalancerRule,
+		ReferenceAttributes: []string{
+			"resource_group_name",
```

Comment:
I think we should add `loadbalancer_id` here and in `lb_rule` resource, as @alikhajeh1 said in details